src/test/ui/issue-53737.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// aux-crate:myfoo=baz.rs
+// edition:2018
+
+
+use myfoo::Yellow;
+
+fn main() {
+    let x = Yellow{};
+}
```

Comment:
Can we port the [extern prelude](https://github.com/rust-lang/rust/tree/master/src/test/run-make-fulldeps/extern-prelude) test to use this?

What about your existing tests for suggestions?